CadRevealFbxProvider.Tests/FbxProviderAttributeParserTests.cs

```diff
@@ -180,10 +185,19 @@ public void ParseFiles_TempScaffTaggedAsWorkorderScaff_ProcessingFails(string cs
         ScaffoldingMetadata metadata;
 
         // assert
-        Assert.Throws<ScaffoldingAttributeParsingException>(() =>
+        var exc = Assert.Catch(() =>
         {
             (var attributes, metadata) = ScaffoldingAttributeParser.ParseAttributes(lines, tempFlag);
         });
+
+        Assert.That(exc, Is.Not.Null);
+
+        Assert.That(
+            exc,
+            Is.InstanceOf<ScaffoldingAttributeParsingException>()
+                .Or.InnerException.InstanceOf<ScaffoldingAttributeParsingException>(),
```

Comment:
Create a helper util for this kind of assert, and use it in all the tests.

Something like an ScaffAssert.ThrowsCustom<ScaffoldingAttributeParsingException>(() => somecode);